src/status_im/contexts/onboarding/enable_notifications/view.cljs

```diff
@@ -20,39 +20,35 @@
     :description                     (i18n/label :t/enable-notifications-sub-title)
     :description-accessibility-label :notifications-sub-title}])
 
+(defn- finish-onboarding
+  []
+  (shell.utils/change-selected-stack-id shell.constants/default-selected-stack true nil)
+  (rf/dispatch [:init-root :shell-stack])
+  (rf/dispatch [:profile/show-testnet-mode-banner-if-enabled])
+  (rf/dispatch [:universal-links/process-stored-event]))
+
+(defn- enable-notifications-and-finish-onboarding
+  []
+  (rf/dispatch [:request-permissions
+                {:permissions [:post-notifications]
+                 :on-allowed  #(log/info "push notification permissions were allowed")
```

Comment:
Not a suggestion for change because code existed before, but both of these log calls should be better placed inside `react-native.permissions/request-permissions` and not be a responsibility of the caller, especially in the view. They are actually debug logs and probably shouldn't pollute the output at the `info` level, more like `debug` level.

Reply:
@ilmotta I just moved the logs to debug, but I didn't want to improve the logging logic since the API is being used in other places and the changes will be too noisy.

I'll improve it as soon as I find room for it :+1: